scripts/overrides/tei_to_html.xsl

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we will need this file now that the other files have been moved to their own repo?